Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:2
+
+RUN apt-get update && apt-get install -y postgresql-client
+
+COPY requirements.txt /usr/src/app/
+WORKDIR /usr/src/app
+RUN pip install -r requirements.txt
+
+EXPOSE 8000
+
+COPY ampcrowd/docker-entrypoint.sh /usr/src/app/ampcrowd
+ENTRYPOINT ["bash", "ampcrowd/docker-entrypoint.sh"]
+CMD ["-d"]
```

Comment:
Agreed. Without the argument, does it enter daemon mode? This would be a bad thing because docker needs a blocking, foreground process or it quits.

What about making it -s or some blocking argument, then providing something like a `dev.sh` which overrides it? Main goal would be to have a runnable script for development.